tools/update-node-report.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Shell script to update ESLint in the source tree to the latest release.
+
+# Depends on npm and node being in $PATH.
+
+# This script must be be in the tools directory when it runs because it uses
+# $BASH_SOURCE[0] to determine directories to work in.
+
+cd "$( dirname "${BASH_SOURCE[0]}" )/../deps/"
+rm -rf node-report
+mkdir node-report-tmp
+cd node-report-tmp
+npm init --yes
+
+npm install --global-style --no-bin-links --production --no-package-lock --ignore-scripts node-report@latest
```

Comment:
This needs to change to pull from GitHub because we don't publish node-report's tests to npm.